
--------------------------- 3D HPS estimation ---------------------------
Traceback (most recent call last):
  File "main.py", line 367, in <module>
    main_worker(args)
  File "main.py", line 287, in main_worker
    train_dataset = dataset.CustomDataset(train_list, image_set="train")
TypeError: __init__() missing 1 required positional argument: 'image_list'
Traceback (most recent call last):
  File "main.py", line 367, in <module>
    main_worker(args)
  File "main.py", line 287, in main_worker
    train_dataset = dataset.CustomDataset(train_list, image_set="train")
TypeError: __init__() missing 1 required positional argument: 'image_list'